swagger.yaml

```diff
@@ -864,4 +897,29 @@ components:
                 enum:
                   - HEAD
                   - TAIL
-
+    SecretSanta:
+      type: object
+      properties:
+        langauge:
+          type: string
+          enum:
+            - es
+            - en
```

Comment:
could we use `es-ES` and `en-GB`? these are the locale types we are using in the frontend

Reply:
ehhhm, isnt `es-ES` specific to spain whilst `es` denotates all Spanish speaking languages? :/ same for en.

Reply:
ah yeah, I though the standard was always 4 letter but it's not. 

btw check this one 😂 
https://github.com/unicode-org/cldr-json/blob/f321d7704452b076a0f75a988889cfdb87802f35/cldr-json/cldr-localenames-full/main/en-GB/languages.json#L178